ApexOSA/filter.py

```
@@ -0,0 +1,271 @@
+        """
+        Return the equipment type and the AP2XXX ID
+        """
+        return "Filter of " + str(self.__ID)
+
+    def GetFilterIdentity(self):
+        """
+        Gets the serial number of the filter board
+        !!! FOR CALIBRATION ONLY !!!
+        """
+
+        Values = []
```

Ruff (0.12.2)

Local variable `Values` is assigned to but never used

Remove assignment to unused variable `Values`

(F841)